drracket-test/tests/drracket/syncheck-test.rkt

```diff
@@ -1770,29 +1770,28 @@
                (copy-port in-port out-port)))))
        (fire-up-drracket-and-run-tests
         (λ ()
-          (let ([drs (wait-for-drracket-frame)])
-            ;(set-language-level! (list "Pretty Big"))
-            (begin
-              (set-language-level! (list "Pretty Big") #f)
-              (test:set-radio-box-item! "No debugging or profiling")
-              (let ([f (test:get-active-top-level-window)])
-                (test:button-push "OK")
-                (wait-for-new-frame f)))
-            (do-execute drs)
-            (let* ([defs (queue-callback/res (λ () (send drs get-definitions-text)))]
-                   [filename (make-temporary-file "syncheck-test~a" #f temp-dir)])
-              (queue-callback/res (λ () (send defs save-file filename)))
-              (preferences:set 'framework:coloring-active #f)
-              (close-the-error-window-test drs)
-              (for-each (run-one-test temp-dir) tests)
-              (preferences:set 'framework:coloring-active #t)
-              (queue-callback/res
-               (λ () 
-                 (send defs save-file) ;; clear out autosave
-                 (send defs set-filename #f)))
-              (delete-file filename)
-             
-              (printf "Ran ~a tests.\n" total-tests-run))))))
+          (define drs (wait-for-drracket-frame))
+          ;(set-language-level! (list "Pretty Big"))
+          (begin
+            (set-language-level! (list "Pretty Big") #f)
+            (test:set-radio-box-item! "No debugging or profiling")
+            (let ([f (test:get-active-top-level-window)])
+              (test:button-push "OK")
+              (wait-for-new-frame f)))
```

Comment:
Resyntax missed an opportunity to inline this `begin` form. Fixing in jackfirth/resyntax#455.